tests.cmake

```diff
@@ -76,14 +76,6 @@ MACRO(BUILD_TEST FILE_NAME)
 
   # Link against Boost.
   TARGET_LINK_LIBRARIES(${EXE_NAME}${PROGRAM_SUFFIX} ${Boost_LIBRARIES})
-
-  # Make sure the plugins will be found.
-  SET_PROPERTY(
-    TEST ${EXE_NAME}${PROGRAM_SUFFIX} PROPERTY ENVIRONMENT
-    "LTDL_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src:$ENV{LTDL_LIBRARY_PATH}")
-  SET_PROPERTY(
-    TEST ${EXE_NAME}${PROGRAM_SUFFIX} PROPERTY ENVIRONMENT
-    "LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src:$ENV{LD_LIBRARY_PATH}")
```

Comment:
Unless I'm missing something, we still need them on Linux, cf. what we did for roboptim-core (`WIN32` check):

```cmake
IF(NOT WIN32)
  SET_PROPERTY(
    TEST ${NAME} PROPERTY ENVIRONMENT
    "LTDL_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src:$ENV{LTDL_LIBRARY_PATH} "
    "LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src/:$ENV{LD_LIBRARY_PATH}")
ENDIF(NOT WIN32)
```